src/pages/EarnTri/EarnTri.tsx

```diff
@@ -8,9 +8,13 @@ import { PoolSection, DataRow } from './EarnTri.styles'
 import EarnTriSortAndFilterContainer from '../../components/EarnTriSortAndFilter/EarnTriSortAndFilterContainer'
 import useFarmsSortAndFilter from './useFarmsSortAndFilter'
 
-const POOLS_ORDER = [32, 33, 5, 11, 31, 8, 30, 7, 0, 3, 4, 15, 19, 20, 23, 24, 26, 27, 28, 29, 34, 36, 37, 38]
-const LEGACY_POOLS = [1, 2, 6, 16, 12, 13, 14, 9, 10, 21, 22, 17, 18]
-const STABLE_POOLS = [35]
+import {
+  DUAL_REWARDS_POOLS,
+  TRI_ONLY_REWARDS_POOLS,
+  ECOSYSTEM_POOLS,
+  STABLE_POOLS,
+  LEGACY_POOLS
+} from '../../constants/farms'
```

Comment:
Instead of importing this, then using them in `useFarmsSortAndFilter`, should we just import them in `src/pages/EarnTri/useFarmsSortAndFilter.tsx`?
```suggestion
```